lib/rubocop/cop/rspec/before_after_all.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that before/after(:all) isn't being used.
+      #
+      # @example
+      #   # bad
+      #   #
+      #   # Faster but risk of state leaking between examples
+      #   #
+      #   describe MyClass do
+      #     before(:all) { Widget.create }
+      #     after(:all) { Widget.delete_all }
```

Comment:
For `around(:each)`, yes, that should be covered too.

I don't think `around(:all)` exists yet: https://github.com/rspec/rspec-core/issues/1031#issuecomment-189409723